learningequality/ka-lite
# Initial support for dubbed videos

This depends on #1101, and fully addresses #1103; please see that issue for changes.
Proper compare is here:
https://github.com/bcipolli/ka-lite/compare/better_topic_caching...dubbed_video_support

The UI is not done, but is currently blocked from progress until an issue with KA can be solved (soon!).  Meanwhile, support can be tested.
- Now, the pre-video topic page should:
  - Show availability, based on any language having a video available.
  - Show you all available videos (in red), and all available subtitles (in blue).
- When you visit the video page,
  - It'll show you the english language videos (hard-coded until the above can be solved)
  - You should have all subtitles available.

How to test:
- Download an english video
- Find a language for which there's a dubbed video youtube ID, and copy the video file and rename the copy to that youtube ID.
- Go to the pre-video topic page, and you should see both languages showing in red.  Note there's a bug such that subtitles will show twice; I'll address that in a UI-only PR, once the above is solved.

![image](https://f.cloud.github.com/assets/4072455/1607067/1090febe-546d-11e3-8f2a-40af90ab833f.png)
